documentation/DCP-documentation/costs.md

```diff
@@ -2,18 +2,24 @@
 
 Distributed-CellProfiler is run by a series of three commands, only one of which incurs costs at typical scale of usage:
 
-[`setup`](step_1_configuration.md) creates a queue in SQS and a cluster, service, and task definition in ECS. 
-ECS is entirely free. 
+[`setup`](step_1_configuration.md) creates a queue in SQS and a cluster, service, and task definition in ECS.
+ECS is entirely free.
 SQS queues are free to create and use up to 1 million requests/month.
 
 [`submitJobs`](step_2_submit_jobs.md) places messages in the SQS queue which is free (under 1 million requests/month).
 
-[`startCluster`](step_3_start_cluster.md) is the only command that incurs costs with initiation of your spot fleet request, creating machine alarms, and optionally creating a run dashboard. 
+[`startCluster`](step_3_start_cluster.md) is the only command that incurs costs with initiation of your spot fleet request, creating machine alarms, and optionally creating a run dashboard.
 
-The spot fleet is the major cost of running Distributed-CellProfiler, exact pricing of which depends on the number of machines, type of machines, and duration of use. 
+The spot fleet is the major cost of running Distributed-CellProfiler, exact pricing of which depends on the number of machines, type of machines, and duration of use.
 Your bid is configured in the [config file](step_1_configuration.md).
+Simple spot fleet configurations can be minimized by:
+
+1) Optimize `MACHINE_TYPE` and `EBS_VOL_SIZE` based on the actual memory and harddrive needs of your run.
+2) When possible, mount your S3 bucket using S3FS so that you can set `DOWNLOAD_FILES = 'False'` to not incur file egress costs.
+3) Set `ASSIGN_IP = 'False'` so that you don't pay for IPv4 addresses per EC2 instance in your spot fleet.
```

Comment:
Can we add a tiny bit more detail here, approximate charges, what this is for, etc? Fine to just point to other points in the docs